manager/manifests/prometheus-dcgm-exporter.yaml.j2

```diff
@@ -0,0 +1,141 @@
+# Copyright 2021 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: monitoring
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dcgm-exporter
+  namespace: default
+  labels:
+    app.kubernetes.io/name: dcgm-exporter
+    app.kubernetes.io/instance: dcgm-exporter
+    app.kubernetes.io/component: dcgm-exporter
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: dcgm-exporter
+  namespace: default
+  labels:
+    app.kubernetes.io/name: dcgm-exporter
+    app.kubernetes.io/instance: dcgm-exporter
+    app.kubernetes.io/component: dcgm-exporter
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: dcgm-exporter
+      app.kubernetes.io/instance: dcgm-exporter
+      app.kubernetes.io/component: dcgm-exporter
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: dcgm-exporter
+        app.kubernetes.io/instance: dcgm-exporter
+        app.kubernetes.io/component: dcgm-exporter
+    spec:
+      serviceAccountName: dcgm-exporter
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              {% if env['CORTEX_PROVIDER'] == "aws" %}
+              - key: k8s.amazonaws.com/accelerator
+                operator: Exists
+              {% elif env['CORTEX_PROVIDER'] == "gcp" %}
+              - key: cloud.google.com/gke-accelerator
+                operator: Exists
+              {% endif %}
```

Comment:
nit: use `config['provider']` instead. This will make it easy to apply the manifest for development 

e.g.

```
python manager/render_template.py dev/config/cluster-aws.yaml manager/manifests/prometheus-dcgm-exporter.yaml.j2 | k apply -f -
```

Reply:
in any case, you do not need the if statement, just check for the label `nvidia.com/gpu`